tests/tensor/test_dist_tensor.py

```diff
@@ -7,8 +7,8 @@
 from torch.testing._internal.common_utils import (
```

Comment:
nit: `test_dist_tensor.py` -> `test_tensor.py`?

Reply:
I agree with @mrshenli  on this renaming. Is it a good idea that we merge this PR and file new PRs for further fixings? My intention is to accelerate the life-cycle of each PR. But I am totally open to other ideas.

Reply:
yep, SGTM